docs/custom-integrations.md

```diff
@@ -0,0 +1,26 @@
+# Custom Integrations
+
+SQL Tasks may be configured to accept an input parameter. This allows other
+extensions to execute SQL Tasks as part of their inner workings, for
+example to allow administrators to implement custom business logic when
+certain events are triggered in their extension.
+
+!!! warning
+    It is necessary to select the "Require user input" task option to
+    enable this feature for a task.
+
+SQL Tasks can be executed using the `Sqltask.execute` API3 endpoint. The
+endpoint accepts two parameters that are relevant for this use-case:
+
+- `task_id`: The ID of the task to be executed
+- `input_val`: The input parameter that will be provided to the task.
+  The value can be an arbitrary data type depending on the use-case;
+  you may want to provide a single ID or a complex JSON object for
```

Comment:
TODO: add a reference to `civirules.md`, where we should describe how to parse complex JSON inputs in SQL (and how to access the input values in different actions in general).